Makefile

```diff
@@ -179,11 +179,12 @@ sbom: sbom/kots-sbom.tgz
 .PHONY: scan
 scan: 
 	trivy fs \
-		--security-checks vuln \
+		--scanners=vuln \
 		--exit-code=1 \
 		--severity="CRITICAL,HIGH,MEDIUM" \
 		--ignore-unfixed \
 		--skip-dirs .github \
+		--skip-dirs web/node_modules \
```

Comment:
maybe have two scans, one for go one for web? i like having this change since it would not pollute the scans with any irrelevant node.js issues, but probably good if we don't completely ignore all js problems.  what do you think?

Reply:
>probably good if we don't completely ignore all js problems.

splititng it would be hard to scan as we are scanning the entire `file system` 
today, we are explicitly ignoring web vulns `--skip-files web/yarn.lock`  [there is a SC to fix these scan results] `web/node_modules` are downloaded libs which wouldn't result in the final build, they can be ignored safely[similar to vendor modules]

Reply:
i guess that was my question.  _why_ are we ignoring web vulns?  some of those dependencies will end up in the transpiled assets.  it's not really related to this PR at all, so maybe we can remove this and discuss in a separate thread?